From: Jubin John <jubin.john@intel.com>
Subject: staging/rdma/hfi1: Remove blank line before close brace
Git-commit: 5161fc3ef60260343c2ffc1b42c9a92ba954d846
Patch-mainline: v4.6
References: bsc#973818, fate#319242
Acked-by: Jiri Bohac <jbohac@suse.cz>

Remove extra blank line before close brace to fix checkpatch check:
CHECK: Blank lines aren't necessary before a close brace '}'

Reviewed-by: Dennis Dalessandro <dennis.dalessandro@intel.com>
Reviewed-by: Ira Weiny <ira.weiny@intel.com>
Reviewed-by: Mike Marciniszyn <mike.marciniszyn@intel.com>
Signed-off-by: Jubin John <jubin.john@intel.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
---
 drivers/staging/rdma/hfi1/chip.c      |    1 -
 drivers/staging/rdma/hfi1/driver.c    |    1 -
 drivers/staging/rdma/hfi1/intr.c      |    1 -
 drivers/staging/rdma/hfi1/mad.c       |    2 --
 drivers/staging/rdma/hfi1/pcie.c      |    1 -
 drivers/staging/rdma/hfi1/sysfs.c     |    1 -
 drivers/staging/rdma/hfi1/uc.c        |    1 -
 drivers/staging/rdma/hfi1/ud.c        |    2 --
 drivers/staging/rdma/hfi1/user_sdma.c |    2 --
 9 files changed, 12 deletions(-)

--- a/drivers/staging/rdma/hfi1/chip.c
+++ b/drivers/staging/rdma/hfi1/chip.c
@@ -13707,7 +13707,6 @@ static void assign_cm_au_table(struct hf
 			SEND_CM_LOCAL_AU_TABLE4_TO7_LOCAL_AU_TABLE6_SHIFT
 		| 64ull * cu <<
 			SEND_CM_LOCAL_AU_TABLE4_TO7_LOCAL_AU_TABLE7_SHIFT);
-
 }
 
 static void assign_local_cm_au_table(struct hfi1_devdata *dd, u8 vcu)
--- a/drivers/staging/rdma/hfi1/driver.c
+++ b/drivers/staging/rdma/hfi1/driver.c
@@ -762,7 +762,6 @@ static inline void finish_packet(struct
 	 */
 	update_usrhead(packet->rcd, packet->rcd->head, packet->updegr,
 		       packet->etail, rcv_intr_dynamic, packet->numpkt);
-
 }
 
 static inline void process_rcv_qp_work(struct hfi1_packet *packet)
--- a/drivers/staging/rdma/hfi1/intr.c
+++ b/drivers/staging/rdma/hfi1/intr.c
@@ -178,7 +178,6 @@ void handle_linkup_change(struct hfi1_de
 		/* notify IB of the link change */
 		signal_ib_event(ppd, ev);
 	}
-
 }
 
 /*
--- a/drivers/staging/rdma/hfi1/mad.c
+++ b/drivers/staging/rdma/hfi1/mad.c
@@ -3805,7 +3805,6 @@ static int subn_get_opa_aggregate(struct
 			return reply((struct ib_mad_hdr *)smp);
 		}
 		next_smp += agg_size;
-
 	}
 
 	return reply((struct ib_mad_hdr *)smp);
@@ -3849,7 +3848,6 @@ static int subn_set_opa_aggregate(struct
 			return reply((struct ib_mad_hdr *)smp);
 		}
 		next_smp += agg_size;
-
 	}
 
 	return reply((struct ib_mad_hdr *)smp);
--- a/drivers/staging/rdma/hfi1/pcie.c
+++ b/drivers/staging/rdma/hfi1/pcie.c
@@ -314,7 +314,6 @@ do_intx:
 		   nvec, ret);
 	*msixcnt = 0;
 	hfi1_enable_intx(dd->pcidev);
-
 }
 
 /* return the PCIe link speed from the given link status */
--- a/drivers/staging/rdma/hfi1/sysfs.c
+++ b/drivers/staging/rdma/hfi1/sysfs.c
@@ -568,7 +568,6 @@ static ssize_t show_serial(struct device
 	struct hfi1_devdata *dd = dd_from_dev(dev);
 
 	return scnprintf(buf, PAGE_SIZE, "%s", dd->serial);
-
 }
 
 static ssize_t store_chip_reset(struct device *device,
--- a/drivers/staging/rdma/hfi1/uc.c
+++ b/drivers/staging/rdma/hfi1/uc.c
@@ -604,5 +604,4 @@ drop:
 op_err:
 	hfi1_rc_error(qp, IB_WC_LOC_QP_OP_ERR);
 	return;
-
 }
--- a/drivers/staging/rdma/hfi1/ud.c
+++ b/drivers/staging/rdma/hfi1/ud.c
@@ -772,7 +772,6 @@ void hfi1_ud_rcv(struct hfi1_packet *pac
 			mgmt_pkey_idx = hfi1_lookup_pkey_idx(ibp, pkey);
 			if (mgmt_pkey_idx < 0)
 				goto drop;
-
 		}
 		if (unlikely(qkey != qp->qkey)) {
 			hfi1_bad_pqkey(ibp, OPA_TRAP_BAD_Q_KEY, qkey,
@@ -810,7 +809,6 @@ void hfi1_ud_rcv(struct hfi1_packet *pac
 		mgmt_pkey_idx = hfi1_lookup_pkey_idx(ibp, pkey);
 		if (mgmt_pkey_idx < 0)
 			goto drop;
-
 	}
 
 	if (qp->ibqp.qp_num > 1 &&
--- a/drivers/staging/rdma/hfi1/user_sdma.c
+++ b/drivers/staging/rdma/hfi1/user_sdma.c
@@ -710,7 +710,6 @@ int hfi1_user_sdma_process_request(struc
 				msecs_to_jiffies(
 					SDMA_IOWAIT_TIMEOUT));
 		}
-
 	}
 	*count += idx;
 	return 0;
@@ -1191,7 +1190,6 @@ static int set_txreq_header(struct user_
 		if (ret)
 			return ret;
 		goto done;
-
 	}
 
 	hdr->bth[2] = cpu_to_be32(
